app/scripts/modules/titus/src/instance/details/instance.details.controller.js

```diff
@@ -118,6 +118,7 @@ module(TITUS_INSTANCE_DETAILS_INSTANCE_DETAILS_CONTROLLER, [
             $scope.instance.externalIpAddress = $scope.instance.placement.host;
             getBastionAddressForAccount(accountDetails, region);
             $scope.instance.titusUiEndpoint = this.titusUiEndpoint;
+            addIpv6Addresses(accountDetails.awsAccount, region, instanceDetails.agentId);
```

Comment:
let's mix this into the promise chain, instead of mutating the object on $scope after the overall promise resolves.